_sass/blocks/revenues/_revenues_subpage_nav.scss

```diff
@@ -7,34 +7,31 @@
 
 
 .revenues_subpage-nav {
-  padding-left: $base-padding;
-  padding-right: $base-padding;
   @include respond-to(medium-up) {
     padding-left: 0;
     padding-right: 0;
   }
-  
+
+  padding-left: $base-padding;
```

Comment:
Expected item on line 15 to appear before line 10. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets